src/mod/externals/FlagWork_Enums.h

```diff
@@ -1770,7 +1770,12 @@ enum class FlagWork_Flag : int32_t {
     FLAG_TRAINER_DOUBLE = 2196,
     FLAG_DISABLE_LEVEL_CAP = 2197,
     FLAG_DISABLE_EXP_SHARE = 2198,
-    FLAG_AFFECTION_ENABLED = 2199,
```

Comment:
For some reason your renaming of this flag got reverted

Reply:
That was a conscious decision. I decided it was better consistency to align all settings flags to the same bool logic to prevent mistakes since there's about 3 seperate calls of FlagWork::GetFlag() in which some are ! flipped etc.